cockatrice/src/window_main.cpp

```diff
@@ -932,9 +932,9 @@ void MainWindow::createTrayIcon()
 void MainWindow::iconActivated(QSystemTrayIcon::ActivationReason reason)
 {
     if (reason == QSystemTrayIcon::DoubleClick) {
-        if (windowState() != Qt::WindowMinimized && windowState() != Qt::WindowMinimized + Qt::WindowMaximized)
+        if ((windowState() & Qt::WindowMinimized) == 0) {
```

Comment:
need to check if this functions the same on multiple oses

Reply:
Feel free to check, but I think it should be fine

Reply:
it's not for me, but it never was outside of windows afaik, so this is out of scope